### ISSUE TITLE
Capturing of a Great general

### ISSUE THREAD

**Issue by forteller** [yairm210/Unciv#5689]
**Platform**
Android 8.0

**Version**
F-droid, 3.18.1

**Describe the bug**
Hey. Thank you for an amazing game!

I noticed that in said in the Civilopedia (shouldn't it be Uncivilopedia, btw? :D) that Great Generals are "uncapturable". So I went exploring with my GG, and they promptly got captured by a barbarian. Bummer! So hopefully you could either update the info, or this is a bug and barbarians should not have been able to capture my GG.

**To Reproduce**
1. Get a Great General
2. Wander around
3. Get captured

**Expected behavior**
GG should not be captured, or Civilopedia should not say they are uncapturable.

Thank you!

**Comment by will-ca** [comment]
@forteller They're uncapturable in that they can't be stolen, I believe. This mirrors Civ V behaviour. Anyone that captures them just gets a worker instead.

Also, keep in mind uncapturable ≠ unkillable or invincible. That Great General was probably toast either way.

**Comment by yairm210** [comment]
Not a bug, as stated, uncapturable means "by other players" not "by barbarians"

**Comment by forteller** [comment]
Why shouldn't this be stated clearly in the Civilopedia, though? I misunderstood, I'm sure others do too.

**Comment by will-ca** [comment]
@forteller What is your suggested description? IMO un*capturable* is accurate: Can't be taken over, but can still be killed or incapacitated. Anything more explicitly precise would be too long.

Also, I haven't looked much at this part of the game code, but I assume the text displayed for Uniques (properties like this) is the same as the text used to define them. So changing it would either break save/mod backwards compatibility or require special-casing to migrate old files.

Probably not worth it for something that only some players would run into, and even then only once. Hm. Though that does bring up another possible solution. Will open a new issue.

**Comment by SomeTroglodyte** [comment]
> text displayed for Uniques (properties like this) is the same as the text used to define them

Precisely. Which leaves the option to leave better display strings in the translations (would work even for `English.properties`, though by convention we don't do that), or change the english "key" in code and invalidate existing translations, or flag the operative unique as non-displayable and add another one with no technical effect...

**Comment by forteller** [comment]
Other option would be to change the notification from "Great general captured by …" to "Great general killed by …"
